.reuse/dep5

```diff
@@ -60,6 +60,10 @@ Files: clients/github-graphql/src/main/assets/*
 Copyright: 2021 Bosch.IO GmbH <osm@bosch.com>
 License: Apache-2.0
 
+Files: integrations/schemas/*
```

Comment:
Commit message nits:
- I think it's fine / better to drop the "JSON-schema" prefix; we usually use these prefixes to denote which *existing* code is touched, but here the code does not exist yet. And the part after ":" also talks about "JSON schema".
- - [3] links to RubyMine docs. Probably https://www.jetbrains.com/help/idea/json.html#ws_json_schema_add_custom is better.